test/inspector_service_test.dart

```diff
@@ -228,11 +199,16 @@ void main() {
 
         await group.dispose();
 
-        await tearDownEnvironment();
+        await env.tearDownEnvironment();
       });
 
       test('render tree', () async {
-        await setupEnvironment(false);
+        await env.setupEnvironment(
```

Comment:
even though it is more boilerplate, specify the config with each test.
That way if someone comments out a test or runs a single test the other tests will still run as expected

Reply:
Added runConfig param for each setupEnvironment call that should not use the default runConfig.